src/accordion/accordion.ts

```diff
@@ -316,7 +316,7 @@ export class NgbAccordion implements AfterContentChecked {
     });
   }
 
-  private _findPanelById(panelId: string): NgbPanel | null { return this.panels.find(p => p.id === panelId); }
+  private _findPanelById(panelId: string): NgbPanel | null { return this.panels.find(p => p.id === panelId) || null; }
```

Comment:
Just to open a discussion on this pattern (I put the comment here, but I found the same in other locations).

This code sounds strange to me, as `| null` has been added in order to resolve types issue. In javascript, null and undefined are often considered the same (that's why a special comparison rule for `== null` or `!= null` is usually enabled for the linter).

On the other hand, I understand that it's not easy to managed different functions which can accept null or undefined. 

So the question : would it be possible (and convenient) to define a custom type (for example : NotDefined: null | undefined) and use it when necessary ?